recipes-bsp/k3-rti-wdt/files/rules

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/make -f
+
+export CC=gcc
```

Comment:
Never set the compiler explicitly. This is done by the dephelper.

Reply:
We are building firmware here, and that does not follow the host CPU rules.

Reply:
Ok, fine.